uitools/import/Esri/ArcGISRuntime/Toolkit/Callout.qml

```diff
@@ -209,143 +161,152 @@ Item {
         CalloutData is obtained from the MapView.
     */
     property var calloutData: null
+    onCalloutDataChanged: {
+        if (calloutData === null) {
+            dismiss();
+        } else if (calloutData.visible) {
+            showCallout();
+        }
+    }
+
+    /*!
+      When \c true, the width of the callout content automatically resizes up
```

Comment:
❓ Not certain, but I believe doc'ing in QML requires you to specify the \brief command (this was the case when we initially wrote the doc for this). Can you confirm please?

Reply:
I'm not seeing any difference in the documented output, nor am I seeing any warnings. Perhaps this requirement has been relaxed? 

Either way, this is a good catch and there's no harm in adding them back so I'll do so.